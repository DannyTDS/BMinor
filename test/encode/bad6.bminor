"This is a 256 byte string AberrationAberrationAberrationAberrationAberrationAberrationAberrationAberrationAberrationAberrationAberrationAberrationAberrationAberrationAberrationAberrationAberrationAberrationAberrationAberrationAberrationAberrationAberration"