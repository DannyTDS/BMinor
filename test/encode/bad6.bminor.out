Cannot decode string "This is a 256 byte string AberrationAberrationAberrationAberrationAberrationAberrationAberrationAberrationAberrationAberrationAberrationAberrationAberrationAberrationAberrationAberrationAberrationAberrationAberrationAberrationAberrationAberrationAberration".
